.github/workflows/test_rust.yml

```diff
@@ -59,7 +59,7 @@ jobs:
         if: runner.os == 'Linux'
         run: |
           sudo apt-get update
-          sudo apt install -y libasound2-dev libxcb-shape0-dev libxcb-xfixes0-dev libgtk-3-dev mesa-vulkan-drivers
+          sudo apt install -y libasound2-dev libxcb-shape0-dev libxcb-xfixes0-dev libgtk-3-dev mesa-vulkan-drivers  libpango1.0-dev
```

Comment:
nit: Is there a reason there are two spaces between "mesa-vulkan-drivers" and "libpango1.0-dev"?

Reply:
Nope, just a typo